src/index.html

```diff
@@ -2,10 +2,17 @@
 	<head>
 		<title>Lithium Onboarding</title>
 		<script src="js/data.js"></script>
+		<link rel="stylesheet" type="text/css" href="general.css">
```

Comment:
any particular reason for separating out the css files into 5 different files for a single page?

Reply:
CSS by component, readability for when/if things became too large. For example the timeline css contains a large amount of css.

Reply:
the timeline.css isn't large, please combine the css files